### ISSUE TITLE
Import (Duplicate Key) problem of last saved file

### ISSUE THREAD

**Issue by D-Dev** [failys/cairis#301]
### Expected behaviour
Scenario - Just firing in many assets for different environments. Not too concerned how they link at the moment. Put too may in one environment, so deleted it, (as it would be quicker to re-build it with less) but wasn't sure if the warning was telling me the objects would be deleted or just unassociated. As the assets etc were deleted (oops), I had a back-up copy on hand, and tried to re-import it back in. 

### Actual behaviour
This produced an error suggesting a duplicate key somewhere. I can't see any duplicates in the XML, have re-ran the quick setup script a few times to refresh the DB (otherwise it creates a different error), but still says I have a duplicate, but I can't find it, so not sure if's the DB scripts being funny, or the XML?

### Steps to reproduce the behaviour
Import the attached test model.
ERROR:cairisd:[400] Database error: MySQL error adding tag (id:1062, message: Duplicate entry '432-117' for key 'PRIMARY'
ERROR:cairisd:[500] Unknown error: MySQL error adding tag (id:1062, message: Duplicate entry '432-117' for key 'PRIMARY'
--------------------------------------------------------------------------------
ERROR in app [/usr/local/lib/python2.7/dist-packages/flask/app.py:1560]:
Exception on /api/import/text [POST]
--------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_restful/__init__.py", line 480, in wrapper
    resp = resource(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flask_restful/__init__.py", line 595, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/cairisuser/cairis/cairis/controllers/CImportController.py", line 78, in post
    raise CairisHTTPError(status_code=500,message=str(ex.message),status='Unknown error')
CairisHTTPError: 500 Internal Server Error: None

[MAA46.zip](https://github.com/failys/cairis/files/2384548/MAA46.zip)

### Version of CAIRIS
Current - did a git pull too in the refresh.

**Comment by D-Dev** [comment]
[Workaround] I've tried to rebuild the xml in sections and after many DB refreshes I've got it all working. It seems though, to get the whole thing working I needed to remove all Tags. I haven't used these before, but gave all assets a tag, some of which were the same (e.g. "ICS) as an identifier. Perhaps it was these duplicates that were causing the issue - but the final solution was to remove them. The working comparison is attached - but I since went back and removed some of the excess environment assets in the xml solving my original need/task.
[testbuild.zip](https://github.com/failys/cairis/files/2385142/testbuild.zip)
